src/Gameplay/Tableau.java

```diff
@@ -0,0 +1,134 @@
+package Gameplay;
+
+import Cards.Card;
+import GUI.GamePanel;
+
+import java.awt.*;
+import java.util.*;
+
+public class Tableau extends Pile{
+
+    public Tableau(int x, int y, int initSize) {
+        super(x, y);
+        super.setSize(67, 600);
+        super.setOpaque(false);
+        for (int i =0; i < initSize; i++){
+            push(GamePanel.getDeck().pop());
```

Comment:
If static objects removed from `GamePanel`, pass the `GamePanel` instance to the `Tableau` constructor to get the deck here.